userland/linker.ld

```diff
@@ -2,7 +2,7 @@ ENTRY(_start)
 
 MEMORY {
     FLASH (rx) : ORIGIN = 0x80000000, LENGTH = 0x00080000
-    SRAM (RWX) : ORIGIN = 0x00000000, LENGTH = 0x00002000
+    SRAM (RWX) : ORIGIN = 0x00000000, LENGTH = 0x0008000
```

Comment:
At one point in time I was getting SRAM overflow errors when I tried to compile the cxx_hello application with 8K of RAM. Increasing to 32K fixed the problem at the time, however I'm having trouble replicating the error now. cxx_hello currently isn't even close to using 8K of RAM.

Since I can't replicate the error, it seems like there isn't a good reason to keep this commit.